src/server/listen.js

```diff
@@ -42,6 +42,7 @@ function listenHttps(app, cb) {
   const serverOptions = {
     key: fs.readFileSync(process.env.HTTPS_PRIVATE_KEY_PATH),
     cert: fs.readFileSync(process.env.HTTPS_PUBLIC_CERT_CHAIN_PATH),
+    minVersion: 'TLSv1.2',
```

Comment:
if you have any version greater than `TLSv1.2`  will that be supported?

Reply:
could this be considered breaking ?

Reply:
@nellyk this is setting the min version https://nodejs.org/dist/latest-v12.x/docs/api/tls.html#tls_tls_createsecurecontext_options

Reply:
thank you, that helped 😄  , could be a breaking change though?
> It is not recommended to use less than TLSv1.2, but it may be required for interoperability.

Reply:
@10xLaCroixDrinker said that we would not be considering this a breaking change